gulpfile.js

```diff
@@ -52,7 +52,11 @@ gulp.task('hbs', function() {
     "title": "IRC Bed Management",
     "headerClass": "with-proposition",
     "assetPath": "",
-    "cookieMessage": "Oooooh! COOKIES!"
+    "cookieMessage": "This website uses cookies.",
+    "availabilityPath": "",
+    "availabilityLink": "Bed availability",
+    "statisticsPath": "#statistics",
```

Comment:
this is in the router, so put it in the scope if you need it from the initialize.
the `gulp` file is not the place to have display content defined, its your build pipeline config